noir-projects/aztec-nr/aztec/src/macros/events/mod.nr

```diff
@@ -9,10 +9,16 @@ comptime fn generate_event_interface(s: StructDefinition) -> Quoted {
 
     let event_type_id = compute_event_selector(s);
 
+    let from_field = get_trait_impl_method(
+        quote { crate::protocol_types::abis::event_selector::EventSelector }.as_type(),
+        quote { crate::protocol_types::traits::FromField },
+        quote { from_field },
+    );
+
     quote {
         impl aztec::event::event_interface::EventInterface<$content_len> for $name {
             fn get_event_type_id() -> aztec::protocol_types::abis::event_selector::EventSelector {
-                aztec::protocol_types::abis::event_selector::EventSelector::from_field($event_type_id)
+                $from_field($event_type_id)
```

Comment:
This was creating warnings and bothering me, so I fixed it in true @benesjan style

Reply:
I feel attacked